controllers/utils/base_reconciler.go

```diff
@@ -44,11 +44,11 @@ import (
 // ReconcilerBase is the base set of components we use in k8s reconcilers.
 // These are items that are not copied for each reconciliation request, and must be written to with caution.
 type ReconcilerBase struct {
-	BaseLogger logr.Logger
-	Scheme     *runtime.Scheme
-	K8sClient  client.Client
-	CSClient   cloud.Client
-	Recorder   record.EventRecorder
+	Scheme           *runtime.Scheme
+	K8sClient        client.Client
+	CSClient         cloud.Client
+	Recorder         record.EventRecorder
+	WatchFilterValue string
```

Comment:
what is this new field for?
if it's not used by the base reconciler, can we put it where it's used?